### PR TITLE
Version 2.1.2

### PR DESCRIPTION
**Pull request by ChristophEichhorn**

### CODE REVIEW

File: cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java [hunk 1]
```diff
@@ -35,13 +35,16 @@
     public static final String LOG_PROVIDER = "[SERVLET]";
     public static final String WRAP_RESPONSE_INIT_PARAM = "wrapResponse";
     public static final String WRAP_REQUEST_INIT_PARAM = "wrapRequest";
+    public static final String LOG_REMOTE_IP = "LogRemoteIP";
```
Comment (altenhof) [inline cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java]: What's the purpose of this? It's defined, but never used.

File: sample/manifest.yml [hunk 2]
```diff
@@ -9,3 +9,5 @@ applications:
   path: target/logging-sample-app-2.1.1.war
   env:
     RANDOM_SLEEP: true
+# Set LogRemoteIP: true to activate logging of remote IPs
+    LogRemoteIP: false
```
Comment (altenhof) [inline sample/manifest.yml]: It's not that critical, but I would favor a consistent naming scheme for environment variables. 
In that sense, I would rather go for `LOG_REMOTE_IP`.

File: cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/LogRemoteIPSettings.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.sap.hcp.cf.logging.common.helper.Environment;
+
+public class LogRemoteIPSettings {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogRemoteIPSettings.class);
+
+    private final String logRemoteIPSetting;
```
Comment (altenhof) [inline cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/LogRemoteIPSettings.java]: Is there a specific reason why this is a String? Wouldn't it be sufficient if it were a boolean? I see that Environment has only one method returning a String, but that doesn't necessarily mean that the variable needs to be a String. Or does it?

Comment (ChristophEichhorn) [reply cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/LogRemoteIPSettings.java]: Has been changed to boolean.

File: cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java (hunk 1 above)
Comment (ChristophEichhorn) [reply cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/RequestLoggingFilter.java]: The purpose is to define the key of the environment variable only once. Has been moved to the Environment class to improve clarity.

File: sample/manifest.yml (hunk 2 above)
Comment (ChristophEichhorn) [reply sample/manifest.yml]: Done.

**Pull request merged**